### PR TITLE
Support basic-auth

### PR DESCRIPTION
**Pull request by iagoborges**

### CODE REVIEW

File: src/kubernetes/api/util.clj [hunk 1]
```diff
@@ -27,21 +29,34 @@
        (if (empty? query) "" "?")
        (query-str query)))
 
+(defn- token [username password]
+  (when (and username password)
+    (str username ":" password)))
```
Comment (andrepoleza) [inline src/kubernetes/api/util.clj]: Are you sure about this format? The [documentation says](http://www.http-kit.org/client.html) it requires `:basic-auth` to be a vector, like `{:basic-auth ["user" "pass"]}` 🤔

File: project.clj [hunk 2]
```diff
@@ -1,4 +1,4 @@
-(defproject kubernetes-api "0.3.3-SNAPSHOT"
```
Comment (andrepoleza) [inline project.clj]: The CHANGELOG entry is missing as well...

Comment (iagoborges) [reply project.clj]: Since this is not a project of ours the CHANGELOG doesn't follow our pattern.

File: src/kubernetes/api/util.clj (hunk 1 above)
Comment (iagoborges) [reply src/kubernetes/api/util.clj]: It works. :man_shrugging:

Comment (andrepoleza) [reply src/kubernetes/api/util.clj]: :ata:

**Pull request merged**